src/stores/WelcomeStore.ts

```diff
@@ -199,4 +199,15 @@ export class WelcomeStore {
   public clearRecommendTags() {
     this._recommendTags = [];
   }
+
+  public get enabledSubmitButton() {
```

Comment:
아 지금보니 enabled가 아니라 disabled가 맞겠네요..허허,,,
```suggestion
  public get disabledSubmitButton() {
```